docs/README.md

```diff
@@ -13,7 +13,7 @@ title: VMWare Secrets Manager for Cloud-Native Apps
 layout: post
 permalink: /
 next_url: /docs/community/
-prev_url: /docs/showcase/
```

Comment:
Please keep this as `/docs/showcase` ; it gives like a circular navigation, and jumping to "showcase" by mistake and seeing lots of awesome videos, would be a lucky mistake :) .